src/vs/workbench/contrib/positronNotebook/browser/PositronNotebookEditor.tsx

```diff
@@ -71,6 +71,8 @@ const POSITRON_NOTEBOOK_EDITOR_VIEW_STATE_PREFERENCE_KEY =
 
 
 export class PositronNotebookEditor extends EditorPane {
+	static readonly ID = 'workbench.editor.positronNotebook';
```

Comment:
This was a bit of me just copying vscode before. Clearly makes a lot more sense here!

Reply:
I missed that it created a cyclic dependency, so now moved it to the new `positronNotebookCommon.ts`.